### PR TITLE
Sponsor page fixes

### PR DESCRIPTION
**Pull request by loicmurumba**
Changed the sponsor logos to a bootstrap grid instead of them being of varying sizes

### CODE REVIEW

File: css/sponsors.css [hunk 1]
```diff
@@ -160,3 +172,6 @@ h1 {
     font-weight: 600;
     color: #666;
 }
+h1.col-md-12{
+    padding-bottom: 50px;
+}
```
Comment (STBTony) [inline css/sponsors.css]: new line here @loicmurumba

File: css/sponsors.css [hunk 2]
```diff
@@ -124,15 +133,18 @@ h1 {
...
-
+.col-sm-2 img{
+    height: 100%;
+    width: 100%;
+    max-width: 100%;
+    max-height: 100%;
+}
+.col-sm-2{
+    padding: 5px;
+}
```
Comment (STBTony) [inline css/sponsors.css]: might not be a great idea

**Comment by STBTony** [comment]
@loicmurumba also why is there only a few logos that's using col-sm-2? Would be way more coherent if all of them can be using bootstrap grid system

File: css/sponsors.css (hunk 1 above)
Comment (loicmurumba) [reply css/sponsors.css]: This is to give more space between the grids and the title. I think this change makes the page look less cluttered

File: css/sponsors.css (hunk 2 above)
Comment (loicmurumba) [reply css/sponsors.css]: Small change that is no longer needed; will remove it in next commit to keep it concise

**Comment by loicmurumba** [comment]
The logos inside the sponsorship subtitle were changed; I kept the other ones formats unchanged because they're more significant sponsors and should have their logos in bigger. But I think you're right, they should still be in a grid layout to keep it consistent. I'll fix this in the next commit.

**Comment by STBTony** [comment]
@loicmurumba Awesome!

File: css/sponsors.css (hunk 1 above)
Comment (STBTony) [reply css/sponsors.css]: @loicmurumba oh no it's a standard to leave a new line at the end of a css file. See the red mark at the bottom of the github change page?

Comment (STBTony) [reply css/sponsors.css]: screen shot 2017-06-02 at 3 53 39 pm

Comment (loicmurumba) [reply css/sponsors.css]: Oh, thanks for the heads up I did not know that. Any reason why? Also could you review the changes made in the last commit? I addressed all the concerns here but there may be more minor things

File: sponsors.html [hunk 3]
```diff
@@ -178,141 +178,263 @@ <h1 style="font-size:50px">CURRENT SPONSORS</h1>
             <h1 class="col-md-12">
               Patrons
             </h1>
-            <a class="tile" href="https://uwaterloo.ca/systems-design-engineering/" target="_blank">
-                <img class="img-rounded" src="img/sponsor/uw_systems_design.png"/>
-            </a>
+            <div class="row">
+              <div class="col-md-4 col-md-offset-4">
+              <a class="tile" href="https://uwaterloo.ca/systems-design-engineering/" target="_blank">
```
Comment (STBTony) [inline sponsors.html]: indentation

File: sponsors.html [hunk 4]
```diff
@@ -178,141 +178,263 @@ <h1 style="font-size:50px">CURRENT SPONSORS</h1>
...
+                    <img class="img-rounded" src="img/sponsor/uw_math_endownment_fund.png"/>
+                  </a>
+                </div>
+                <div class= "col-md-2">
+                  <a class="tile" href="http://www.husky.co/EN-US/" target="_blank">
+                    <img class="img-rounded" src="img/sponsor/husky.png"/>
+                  </a>
+                </div>
+          </div>
           </div>
```
Comment (STBTony) [inline sponsors.html]: what's going on here

**Comment by STBTony** [comment]
@loicmurumba  a lot of your new changes are inconsistent with the spacing. It won't affect the html itself it's just nit pick. maybe change your sublime setting and see

File: sponsors.html [hunk 5]
```diff
@@ -178,141 +178,263 @@ <h1 style="font-size:50px">CURRENT SPONSORS</h1>
...
-            </a>
-            <a class="tile" href="http://odg.com/" target="_blank">
-              <img class="img-rounded" src="img/sponsor/ontario_drive_and_gear.png"/>
-            </a>
-            <a class="tile" href="http://www.teslamachining.com/" target="_blank">
-              <img class="img-rounded" src="img/sponsor/tesla_machining_and_gundrilling.png"/>
-            </a>
+            <div class="row">
+
+	            <div class="col-sm-2 col-sm-offset-1">
```
Comment (STBTony) [inline sponsors.html]: here for example. the tab you are using isn't consistent with everywhere else in this file.

**Comment by STBTony** [comment]
Other than that LGTM

File: sponsors.html [hunk 6]
```diff
@@ -178,141 +178,285 @@ <h1 style="font-size:50px">CURRENT SPONSORS</h1>
...
-              <img class="img-rounded" src="img/sponsor/ontario_drive_and_gear.png"/>
-            </a>
-            <a class="tile" href="http://www.teslamachining.com/" target="_blank">
-              <img class="img-rounded" src="img/sponsor/tesla_machining_and_gundrilling.png"/>
-            </a>
+
+            <div class="row">
+
+	            <div class="col-sm-2 col-sm-offset-1">
+		            <a class="tile" href="https://uwaterloo.ca/chemical-engineering/employment-opportunities/faculty" target="_blank">
```
Comment (STBTony) [inline sponsors.html]: @loicmurumba Are you sure you fixed the spacing? Try using sublime's spacing viewer to see if it's actually spaces or tabs. On github it shows pretty obviously but maybe not on sublime

**Review by STBTony — APPROVED** [review]

**Pull request merged**